lib/routes.json

```diff
@@ -394,6 +394,92 @@
                 "invalidmsg": "",
                 "description": "OAuth token"
             }
+        },
+        "acceptTree": {
+            "application/vnd.github.mockingbird-preview": [
+                "/repos/:owner/:repo/issues/:issue_number/timeline"
+            ],
+
+            "application/vnd.github.spiderman-preview": [
+                "/repos/:owner/:repo/traffic/popular/referrers",
+                "/repos/:owner/:repo/traffic/popular/paths",
+                "/repos/:owner/:repo/traffic/views",
+                "/repos/:owner/:repo/traffic/clones"
+            ],
+
+            "application/vnd.github.squirrel-girl-preview": [
+                "/repos/:owner/:repo/comments/:id/reactions",
+                "/repos/:owner/:repo/issues/:number/reactions",
+                "/repos/:owner/:repo/issues/comments/:id/reactions",
+                "/repos/:owner/:repo/pulls/comments/:id/reactions",
+                "/reactions/:id"
+            ],
+
+            "application/vnd.github.polaris-preview": [
+                "/repos/:owner/:repo/pulls/:number/merge"
+            ],
+
+            "application/vnd.github.loki-preview+json": [
+                "/repos/:owner/:repo/branches",
+                "/repos/:owner/:repo/branches/:branch",
+                "/repos/:owner/:repo/branches/:branch/protection"
+            ],
```

Comment:
A few more routes to add:

/repos/:user/:repo/branches/:branch/protection/required_status_checks
/repos/:user/:repo/branches/:branch/protection/required_status_checks/contexts
/repos/:user/:repo/branches/:branch/protection/restrictions
/repos/:user/:repo/branches/:branch/protection/restrictions/teams
/repos/:user/:repo/branches/:branch/protection/restrictions/users